test/support/docker_wrapper.rb

```diff
@@ -0,0 +1,50 @@
+Minitest.after_run do
+  DockerWrapper.stop if DockerWrapper.running?
+end
+
+module DockerWrapper
```

Comment:
🗒️ This code was mostly lifted from https://github.com/capistrano/capistrano/pull/2159.